backend-rust/src/indexer.rs

```diff
@@ -1340,3 +1380,134 @@ impl PreparedBakerEvent {
         Ok(())
     }
 }
+
+struct PreparedModuleDeployed {
```

Comment:
Sorry I'm a bit out of the loop here, what does it mean to "prepare" an event? It sounds like some kind of specific term.

Reply:
`Prepared` means that the data is ready to be inserted into the database (types have been converted to postgres data typs).